File: C#/Easy/Flipping an Image.py
class Solution:
    def flipAndInvertImage(self, image: List[List[int]]) -> List[List[int]]:
        rows = len(image)
        cols = len(image[0])
        
        for i in range(rows):
            image[i] = list(reversed(image[i]))
            for j in range(cols):
                image[i][j] += 1 if image[i][j] == 0 else -1
        
        return image

File: LeetCode/C#/Easy/Flipping an Image.cs
public class Solution {
    public int[][] FlipAndInvertImage(int[][] image) {
        var result = new List<int[]>();
        foreach(int[] i in image){
            var temp = i.Reverse().ToArray();
            for(int j = 0; j < temp.Length; j++) {
                temp[j] = temp[j] == 1 ? 0 : 1;
            }
            result.Add(temp);
        }
        return result.ToArray();
    }
}